content/02.main-text.md

```diff
@@ -26,7 +26,7 @@ Writing review articles in a public forum allows review authors to engage with t
 In addition, discussing manuscripts in the open provides one form of pre- and post-publication peer review [@doi:10.7554/eLife.38532], incentivizing the reviews with potential manuscript authorship.
 However, inviting wide authorship brings many technical and social challenges such as how to fairly distribute credit, coordinate the scientific content, and collaboratively manage extensive reference lists.
 
-To coordinate this effort, we developed a manuscript writing process using the Markdown language, [GitHub](https://github.com/greenelab/deep-review/), and our new [Manubot](https://github.com/greenelab/manubot-rootstock) tool for automating manuscript generation.
+To coordinate this effort, we developed a manuscript writing process using the Markdown language, [GitHub](https://github.com/greenelab/deep-review), and our new [Manubot](https://github.com/greenelab/manubot-rootstock) tool for automating manuscript generation.
```

Comment:
Is it odd that the GitHub link goes specifically to our review?

Reply:
I removed that link in #54